awx/ui/src/screens/Instances/InstanceList/InstanceListItem.js

```diff
@@ -0,0 +1,248 @@
+import React, { useState, useCallback } from 'react';
+import { Link } from 'react-router-dom';
+import { bool, func } from 'prop-types';
+import { t, Plural } from '@lingui/macro';
+import styled from 'styled-components';
+import 'styled-components/macro';
+import {
+  Progress,
+  ProgressMeasureLocation,
+  ProgressSize,
+  Slider,
+  Tooltip,
+} from '@patternfly/react-core';
+import { Tr, Td, ExpandableRowContent } from '@patternfly/react-table';
+import { formatDateString } from 'util/dates';
+import { ActionsTd, ActionItem } from 'components/PaginatedTable';
+import InstanceToggle from 'components/InstanceToggle';
+import StatusLabel from 'components/StatusLabel';
+import { Instance } from 'types';
+import useRequest, { useDismissableError } from 'hooks/useRequest';
+import useDebounce from 'hooks/useDebounce';
+import { InstancesAPI } from 'api';
+import { useConfig } from 'contexts/Config';
+import AlertModal from 'components/AlertModal';
+import ErrorDetail from 'components/ErrorDetail';
+import { Detail, DetailList } from 'components/DetailList';
+
+const Unavailable = styled.span`
+  color: var(--pf-global--danger-color--200);
+`;
+
+const SliderHolder = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+`;
+
+const SliderForks = styled.div`
+  flex-grow: 1;
+  margin-right: 8px;
+  margin-left: 8px;
+  text-align: center;
+`;
+
+function computeForks(memCapacity, cpuCapacity, selectedCapacityAdjustment) {
```

Comment:
This function could be moved an utils file, since it is even defined outside the components in which is used.